app/javascript/components/patient/assessment/actions/CurrentStatus.js

```diff
@@ -102,7 +102,7 @@ class CurrentStatus extends React.Component {
       return (
         <ReactTooltip id={`symptomatic-non-test-based`} multiline={true} place="top" type="dark" effect="solid" className="tooltip-container">
           <span>
-            At least 10 days have passed since the Symptom Onset Date and at least 24 hours have passed since the case last reported “Yes” for fever or use of
+            At least 5 days have passed since the Symptom Onset Date and at least 24 hours have passed since the case last reported “Yes” for fever or use of
```

Comment:
This value needs to be passed from the back end

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1188/commits/26da7407b9271543f7436c9624ccd29ffc9e5fed

Reply:
thanks for making the update!